### DISCUSSION TITLE
Honor points rewards

### DISCUSSION THREAD

**Discussion by alexisgohan2** [chromiecraft/chromiecraft#4853] [Suggestions]
GM, like a suggestion, please increase the amount of gold that you reward for honor points 50g for 50k is really low for TBC (on vanilla could be good, but no on tbc, everything is more expensive )

**Comment by 55Honey** [comment]
Hi @alexisgohan2 ,
thanks for your suggestion. 
There is a lot of extra gold already and farming gold is very easy. Normally there would be no gold at all for honor points. It's not meant to make anyone rich also.

This is not something we want to look into.

**Reply by alexisgohan2** [reply]
hi honey

i know there are other ways to get gold, but most of them are on open world and pve (killing mobs anyway, not killing players) and no everybody likes the pve... i know that normally there would be no gold for honor points , but normally we could buy gems with it and sell it on auction house to get some gold

if the only way to spend honor is buying mounts i think the pvp isn't profitable until level 80, then

**Reply by alexisgohan2** [reply]
Just to make some random numbers.... in 30 minutes of open world we can make like 100g selling droped items.... while we need to "**WIN**" like 32 bgs to ge that amount of gold :(  

Balancing this would be good

**Reply by 55Honey** [reply]
> Normally there would be no gold at all for honor points.

It's just how 3.3.5 works, unfortunately. There are currently no plans to increase the gains from honor.
